pom.xml

```diff
@@ -54,6 +54,18 @@
 				<artifactId>hamcrest</artifactId>
 				<version>${hamcrest.version}</version>
 			</dependency>
+      <dependency>
+			  <groupId>junit</groupId>
+			  <artifactId>junit</artifactId>
+			  <version>4.12</version>
+			  <scope>test</scope>
+		  </dependency>
+      <dependency>
+			  <groupId>org.mockito</groupId>
+			  <artifactId>mockito-all</artifactId>
+			  <version>1.10.19</version>
```

Comment:
you can add mockito but if you add the latest version of it

Reply:
This is the latest version of the mockito library. Mockito version 2.0.x is still in beta. Here are all dispalyed all of the versions of this library: https://mvnrepository.com/artifact/org.mockito/mockito-all

Reply:
But in another PR, the dependency who is imported is mickito-core 3.5.13, the official website indicate the latest version of mockito is 3.5.12: https://site.mockito.org/, the github has a release on 3.5.13 https://github.com/mockito/mockito. I think mockito-all is no more used, it's mockito-core we should use